libr/core/cmd_debug.c

```diff
@@ -790,7 +804,12 @@ static int step_until_inst(RCore *core, const char *instr, bool regex) {
 		if (r_debug_is_dead (core->dbg)) {
 			break;
 		}
-		r_debug_step (core->dbg, 1);
+		if (is_repeatalbe_inst (core->dbg)) {
+			r_debug_step_over (core->dbg, 1);
+		}
```

Comment:
Wrong indent

Reply:
sorry, maybe I'm too tired, but can't see any indent issue.

Reply:
the "else" must be in the lline above "} else {"